apps/expo-go/android/app/build.gradle

```diff
@@ -219,24 +222,19 @@ dependencies {
   implementation 'de.greenrobot:eventbus:2.4.0'
 
   implementation 'com.squareup.picasso:picasso:2.5.2'
-  implementation 'com.google.android.gms:play-services-analytics:16.0.1'
   implementation 'com.google.android.gms:play-services-maps:18.0.0'
-  implementation 'com.google.android.gms:play-services-auth:15.0.1'
-  implementation 'com.google.android.gms:play-services-location:15.0.1'
+  implementation 'com.google.android.gms:play-services-auth:18.1.0'
+  implementation 'com.google.android.gms:play-services-location:21.0.1'
   debugImplementation 'com.squareup.leakcanary:leakcanary-android-no-op:1.4-beta1'
   // debugImplementation 'com.squareup.leakcanary:leakcanary-android:1.4-beta1'
   releaseImplementation 'com.squareup.leakcanary:leakcanary-android-no-op:1.4-beta1'
   implementation 'com.facebook.device.yearclass:yearclass:2.1.0'
   implementation 'commons-io:commons-io:1.4'
-  implementation 'me.leolin:ShortcutBadger:1.1.4@aar'
-  implementation 'com.github.CanHub:Android-Image-Cropper:1.1.1'
+  implementation 'com.github.CanHub:Android-Image-Cropper:4.3.1'
   implementation 'commons-codec:commons-codec:1.10'
   implementation 'com.google.zxing:core:3.3.3'
   implementation 'net.openid:appauth:0.4.1'
-  implementation 'com.airbnb.android:lottie:3.4.0'
-  implementation('io.nlopez.smartlocation:library:3.2.11') {
-    transitive = false
-  }
```

Comment:
I believe this is used by expo-location, does it still works fine when removing this?

Reply:
I added it back and build is passing; but I cant' tell for now if everything works with / without it bcs I cannot get the app to build.

Reply:
managed to build and test